website/content/docs/platform/k8s/vso/sources/vault/client-cache.mdx

```diff
@@ -0,0 +1,266 @@
+
+For [Helm installs](/vault/docs/platform/k8s/vso/installation#installation-using-helm), the [`server.clientCache`](/vault/docs/platform/k8s/vso/helm#v-controller-manager-clientcache) configuration section has all the options necessary to setup encrypted client cache storage.
+
+Install or upgrade the chart using these values:
+
+```yaml
+controller:
+  manager:
+    clientCache:
+      persistenceModel: direct-encrypted
+      storageEncryption:
+        enabled: true
+        vaultConnectionRef: local-vault-server
+        keyName: vso-client-cache
+        transitMount: transit
+        namespace: <VAULT_NAMESPACE>
+        method: kubernetes
+        mount: kubernetes
+        kubernetes:
+          role: operator
+          serviceAccount: vault-secrets-operator-controller-manager
+          tokenAudiences: ["vault"]
+```
+
+## Step 5: Enable caching in the operator (OLM/OperatorHub)
+
+For [OpenShift OperatorHub](/vault/docs/platform/k8s/vso/openshift#operatorhub) installs, add a VaultAuth configured for storage, and set the `VSO_CLIENT_CACHE_PERSISTENCE_MODEL` environment variable in the operator's subscription to enable encrypted client cache storage.
+
+1. Add a [VaultAuth](/vault/docs/platform/k8s/vso/api-reference#vaultauth) to your cluster with the label `cacheStorageEncryption: 'true'` and [spec.storageEncryption](/vault/docs/platform/k8s/vso/api-reference#vaultauthspec) configured.
+
+  ```yaml
+  apiVersion: secrets.hashicorp.com/v1beta1
+  kind: VaultAuth
+  metadata:
+    name: operator
+    namespace: vault-secrets-operator
+    labels:
+      cacheStorageEncryption: 'true'
+  spec:
+    kubernetes:
+      role: operator
+      serviceAccount: vault-secrets-operator-controller-manager
+      tokenExpirationSeconds: 600
+      audiences: ["vault"]
+    method: kubernetes
+    mount: kubernetes
+    namespace: <VAULT_NAMESPACE>
+    storageEncryption:
+      keyName: vso-client-cache
+      mount: transit
+    vaultConnectionRef: local-vault-server
+  ```
+
+1. With the operator installed through your cluster's OperatorHub, edit the subscription to set the `VSO_CLIENT_CACHE_PERSISTENCE_MODEL` environment variable in the operator's subscription in the web console or with the OpenShift CLI.
+
+  Using the OpenShift web console:
+    - Navigate to Operators -> Installed Operators
+    - Select the "Vault Secrets Operator"
+    - "Edit Subscription" in the top right action menu
+
+  Using the OpenShift CLI:
+
+    ```shell-session
+    oc edit subscription vault-secrets-operator -n vault-secrets-operator
+    ```
+
+  Set the following environment variable at `spec.config.env`:
+
+  ```yaml
+  spec:
+    name: vault-secrets-operator
+    channel: stable
+    config:
+      env:
+        - name: VSO_CLIENT_CACHE_PERSISTENCE_MODEL
+          value: direct-encrypted
+  ```
+
+  The pod in the operator deployment will be restarted once the new environment variable is applied.
+
+## Step 6: Verify the client cache storage is enabled
```

Comment:
nit: Should we mark this as optional? Not sure.

```suggestion
## Optional: Verify the client cache storage is enabled
```

Reply:
Sure that makes sense